app/mixins/components/with-config-validation.js

```diff
@@ -11,9 +11,10 @@ export default Mixin.create({
 
   hasMessages: gt('messages.length', 0),
 
-  isConfigValidationEnabled: and('auth.signedIn', 'repo.settings.config_validation'),
-
-  showConfigValidation: reads('isConfigValidationEnabled'),
+  // we want to show config messages for unauthenticated users, too. however,
+  // we then do not have a way to ask for the repository's config_validation
+  // setting. maybe we can check if the request.messages key is present?
+  showConfigValidation: true,
```

Comment:
hmmmm, i don't think so. we would then not show the validation result on repos where one does not have a membership and there are no messages. e.g. if you look at the Ember repo's builds on Travis CI (assuming you're not a contributor to Ember) and if they have no validation messages, then the validation result would be hidden ... while it should message "everything valid, great job!"

there is also a longer explanation of the issue here: https://github.com/travis-ci/travis-web/pull/2399#issuecomment-604679907, in case you haven't seen it.

what do you think?